hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetImpl.java

```diff
@@ -1049,12 +1050,17 @@ static File moveBlockFiles(Block b, ReplicaInfo replicaInfo, File destdir)
   static File[] hardLinkBlockFiles(ReplicaInfo srcReplica, File dstMeta,
       File dstFile)
       throws IOException {
+    FsVolumeSpi srcReplicaVolume = srcReplica.getVolume();
```

Comment:
Hi @virajjasani here I have one question to ask you.
It seems that there is no change in code logic. Are these codes for adding debug or trace info?

Reply:
That's right, these changes are to simplify using same variable for logging as well as method arguments.
And one more improvement is that before patch, we have isDebugEnabled() guarding info log, this is also fixed with this PR. It's the last change of this file.